### ISSUE TITLE
Attract more contributors to i18n

### ISSUE THREAD

**Issue by Ismael-VC** [JuliaLang/julialang.github.com#428]
@ViralBShah this is related to your comment:
- https://github.com/JuliaLang/julialang.github.com/pull/252#issuecomment-249360804

> Is it possible in the language dropdown to also have something that will invite people to contribute translations?

We could add one of this widgets that show the current status of the project and redirect users that click on it to the julialang-web project at github, we could also change the displayed text of course, to an invitation to participate, this could be added as a footer in the front page?
- https://www.transifex.com/julialang-i18n/julialang-web/widgets

![image](https://cloud.githubusercontent.com/assets/4594825/18861839/c369a442-844d-11e6-8f2f-cdc4898cd533.png)

There are two options a static image (that is updated by transifex) of an interactive chart (not so different to the image at all).

**Comment by ViralBShah** [comment]
I think the ideal option would be to have an extra option called `Contribute` in the transifex drop down - since people who click there for other languages are also the ones likely to be able to contribute. Is the javascript for that customizable?

**Comment by ViralBShah** [comment]
Cc @shashi Our resident javascript expert.

**Comment by waldyrious** [comment]
Considering that the transifex script is a remote one (c.f. 32fd440), we can’t really edit it. We’ll have to add a script snippet directly in this repo.

I can give that a shot, but I'll need to figure out how to detect when the transifex script has completed. I believe script tags (either local or remote) that are unadorned with the [async or the defer attributes](http://www.growingwiththeweb.com/2014/02/async-vs-defer-attributes.html) will execute sequentiually, so just adding the script below the transifex one *should* work. Can anyone confirm?

**Comment by Ismael-VC** [comment]
Please also see this discussion at julia-users:

* https://groups.google.com/d/msg/julia-users/9TmQOC5K7Ek/VCSKow1CAwAJ


@nalimilan 

> I'm worried that we completely
destroy the professional aspect of the website by having random people
do weird things in each language no core developer understands.

I answered:

> I think the best way to approach this is not to be conservative, but to be open and transparent, so we can get more help and others can't be disappointed for the current status, and the reputation of all, not only of the Julia project, but also of the contributors that are willing to translate for us, remains intact and even become more positive.

@ViralBShah do you think we can put the banner I proposed for a short and fixed amount of time perhaps? Any other ideas?

See also the project management board:

* https://github.com/Julia-i18n/julia-i18n/projects/1

The least thing I want is to have the reputation of anyone affected, what I want is completely the contrary of that. If you don't like my proposal, can you think of another way of achieving this?

Thanks for all your concern and support, cheers!

**Comment by waldyrious** [comment]
> the banner I proposed

I think a banner would be rather intrusive. Publishing only reviewed translations would seem like a good way to reduce translation errors, but the correct fix IMO is to make it clear that anyone can contribute to the translations, and provide an easy way to do so (i.e. this issue!), so that when someone detects a mistake, they can go ahead and fix it directly :)

**Comment by Ismael-VC** [comment]
Then maybe we need a blog post in the official channels and lot of noise about it in the social networks (I already do that)? I've also done some video tutorials:

This one shows the workflow (the first I did):

* https://www.livecoding.tv/ismael-vc/videos/oyvBm-julia-language-website-i18n

This one is a video tutorial + rambling 😸 due to the streaming nature of the video:

* https://www.livecoding.tv/ismael-vc/videos/Or5Gr-julia-language-website-i18n-tutorial

Excuse the unprofessional quality of both the video and my pronunciation, but it will improve them in future iterations.

**Comment by waldyrious** [comment]
(just to clarify, I meant clearer in the website itself, i.e. in context: if someone notices an incorrect translation, the very dropdown where they picked the translation from would have the prompt to contribute themselves -- which should make it clear that the error is easily fixable, and that the translations are indeed contributed by volunteers.)

**Comment by waldyrious** [comment]
Would something like this work? @Ismael-VC @ViralBShah 

![screenshot 2016-10-13 15 19 03](https://cloud.githubusercontent.com/assets/478237/19352673/93d0a5d8-9158-11e6-8c70-366dcbc33819.png)

Any suggestions in terms of color, font, positioning, wording? And where should the link point to? Perhaps https://github.com/Julia-i18n/julia-i18n/blob/master/CONTRIBUTING.md (note: doesn't exist yet)?

**Comment by Ismael-VC** [comment]
@waldyrious OMG how did you made this!? 😄  Can we put it in the staging website in order to test it? How does this work? Thank you for doing this!

**Comment by waldyrious** [comment]
@Ismael-VC that was just a mock-up 😛 I'm waiting on the feedback I requested in that comment before moving on to actually implementing this.

**Comment by ViralBShah** [comment]
Are we going to do something here? Just checking.

**Comment by waldyrious** [comment]
@ViralBShah do you agree with the proposal I made above? If there are no objections, I can make a PR.

**Comment by ViralBShah** [comment]
I am ok with it. Let's try it out.

**Comment by waldyrious** [comment]
Ok. To start out, I made some CSS adjustments to the Transifex menu in #551. Afterwards I'll send a separate PR adding a script to dynamically append the link to the dropdown.

**Comment by Ismael-VC** [comment]
I don't want to keep dealing with the Julia stewards or have any connection with them. I will make a page in Spanish, I don't need this anymore, If anyone is interested feel free to continue this.
